docker-compose.yaml

```diff
@@ -6,8 +6,8 @@ services:
     networks:
       - internal-network
     volumes:
-      - cardano-node-data:/data
-      - cardano-node-ipc:/ipc
+      - ./cardano-node-data:/data
```

Comment:
These reference the volumes defined below, not a local folder

Reply:
Thanks, I was losing that!